tests/cloner_test.go

```diff
@@ -120,3 +124,19 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func cloneOfAnnoExistenceTest(f *framework.Framework, targetNamespaceName string) {
+	// Create targetPvc
+	By("Creating target pvc")
```

Comment:
Please append namespace to this text. Standard format, if you don't want two separate inserts is:
   _pvcNamespace/pvcName_